ep016/README.md

```diff
@@ -0,0 +1,44 @@
+# [pretty python profiling (intermediate)](https://youtu.be/ey_P64E34g0)
+
+Today I show how I figure out what's slow in a python application and some cool visualization tools!
```

Comment:
Do you want the extra info from the video description to be here as well?

Reply:
up to you

Reply:
Links are subject that can change in the future and they are at two places already - in the video & in the video description, so I guess this is enough redundancy :D